src/hooks/wallets/useOnboard.ts

```diff
@@ -46,12 +46,18 @@ export const getConnectedWallet = (wallets: WalletState[]): ConnectedWallet | nu
   const account = primaryWallet?.accounts[0]
   if (!account) return null
 
-  return {
-    label: primaryWallet.label,
-    address: getAddress(account.address),
-    ens: account.ens?.name,
-    chainId: Number(primaryWallet.chains[0].id).toString(10),
-    provider: primaryWallet.provider,
+  try {
+    const address = getAddress(account.address)
+    return {
+      label: primaryWallet.label,
+      address,
+      ens: account.ens?.name,
+      chainId: Number(primaryWallet.chains[0].id).toString(10),
+      provider: primaryWallet.provider,
+    }
+  } catch (error) {
+    console.error(error)
```

Comment:
We track our errors via Sentry so it would be good to use our logging function here instead. In order to do so, you need add an error message to our [`ErrorCodes`](https://github.com/safe-global/web-core/blob/dev/src/services/exceptions/ErrorCodes.ts) and use `logError` ([example](https://github.com/safe-global/web-core/blob/dev/src/utils/transactions.ts#L186-L188)).

I'd suggest 106, e.g. "106: Failed to get connected wallet".

Reply:
Ok sure, I will work on that